src/atoms/referencesState.ts

```diff
@@ -5,15 +5,11 @@ import { ReferenceItem } from '../types/ReferenceItem';
 
 type ReferencesState = Record<string, ReferenceItem>;
 
-/**
- * INTERNAL ATOMS
- */
+/** INTERNAL ATOMS */
```

Comment:
These got shortened to the one-line form. But really they shouldn't be JSDoc comments at all (they could just be inline comments, or `/* ... */` comments -- not jsdoc).